src/x86/sse41.rs

```diff
@@ -6,6 +6,34 @@ use stdsimd_test::assert_instr;
 
 use v128::*;
 
+// SSE4 rounding constans
+/// round to nearest
+pub const _MM_FROUND_TO_NEAREST_INT: i32 = 0x00;
+/// round down
+pub const _MM_FROUND_TO_NEG_INF: i32 = 0x01;
+/// round up
+pub const _MM_FROUND_TO_POS_INF: i32 = 0x02;
+/// truncate
+pub const _MM_FROUND_TO_ZERO: i32 = 0x03;
+/// use MXCSR.RC; see `vendor::_MM_SET_ROUNDING_MODE`
+pub const _MM_FROUND_CUR_DIRECTION: i32 = 0x04;
+/// do not suppress exceptions
+pub const _MM_FROUND_RAISE_EXC: i32 = 0x00;
+/// suppress exceptions
+pub const _MM_FROUND_NO_EXC: i32 = 0x08;
+/// round to nearest and do not suppress exceptions
+pub const _MM_FROUND_NINT: i32 = (_MM_FROUND_RAISE_EXC | _MM_FROUND_TO_NEAREST_INT);
+/// round down and do not suppress exceptions
+pub const _MM_FROUND_FLOOR: i32 = (_MM_FROUND_RAISE_EXC | _MM_FROUND_TO_NEG_INF);
+/// round up and do not suppress exceptions
+pub const _MM_FROUND_CEIL: i32 = (_MM_FROUND_RAISE_EXC | _MM_FROUND_TO_POS_INF);
+/// truncate and do not suppress exceptions
+pub const _MM_FROUND_TRUNC: i32 = (_MM_FROUND_RAISE_EXC | _MM_FROUND_TO_ZERO);
+/// use MXCSR.RC and do not suppress exceptions; see `vendor::_MM_SET_ROUNDING_MODE`
+pub const _MM_FROUND_RINT: i32 = (_MM_FROUND_RAISE_EXC | _MM_FROUND_CUR_DIRECTION);
+/// use MXCSR.RC and suppress exceptions; see `vendor::_MM_SET_ROUNDING_MODE`
+pub const _MM_FROUND_NEARBYINT: i32 = (_MM_FROUND_NO_EXC | _MM_FROUND_CUR_DIRECTION);
```

Comment:
This one and that one above it have a little bit confusing names. Not sure if I should define these some extra constants (not searchable in Intel Intrinsics Guide), but [clang does](https://clang.llvm.org/doxygen/smmintrin_8h_source.html), so I did.

Reply:
Yeah I think it's ok to match clang in this regard